docs/release_notes/release_notes.md

```diff
@@ -9,67 +9,33 @@ has_children: true
 # Release Notes
 
 
-## Version 4.0.0 (June 2023)
+## Version 4.1.0 (July 2023)
 
-* ### Notable Event Assignment
-    * User can assign notable events to the user on Splunk for further investigation of the incident. User can also update the status of the notable event based on the investigation phase.
-    * Added the new SOC Dashboard to get overview of notable events by the alert, assignee, status and severity.
-    * Updated Overview dashboard to filter the result based on the Status value.
+* ### Added "Network Compromise - DDoS Behavior Detected" new alert.
 
-![alt]({{ site.baseurl }}/assets/notable_event_assignment.png)
+* ### Enhancements  
+    * Improved all the O365 Login related alerts and dashboards to show more information using the O365 Audit Signin Logs.
```

Comment:
move this to a topic in New feature of its own:

Using Office 365 Login data from new Input to Improve Results:
* Improved all the O365 Login-related alerts and dashboards to show more information using the O365 Audit Sign-in Logs from Splunk Add-on for Office 365.